applications/sintering/analysis_examples/49particles_delta_min_40.json

```diff
@@ -64,7 +73,7 @@
         "NewtonReusePreconditioner": false
     },
     "Output": {
-        "Contour": "true",
+        "Contour": "false",
```

Comment:
It just probably makes sense to make similar alterations in the json files for other interface thicknesses (05, 10, 20, 30).